.github/workflows/ci.yml

```diff
@@ -63,8 +63,23 @@ jobs:
           python -m pip install .[test]
           python -m pip install ${{ matrix.backend }}
 
-      - name: Test
-        run: pytest --color=yes --cov=napari_omero --cov-report=xml
+      - name: Set up OMERO
+        if: runner.os == 'Linux'
+        run: |
+          git clone https://github.com/openmicroscopy/omero-test-infra .omero
+          # ensure the ports are bound correctly
+          # see also: https://github.com/ome/omero-test-infra/issues/77
+          echo 'OMERO_SERVER_SSL=4064:' >> .omero/.env
+          echo 'OMERO_WEB_PORT=4080:' >> .omero/.env
+          .omero/compose up -d
+          .omero/wait-on-login
+
+      - name: Test with OMERO server
+        if: runner.os == 'Linux'
+        run: pytest ./tests/server --color=yes --cov=napari_omero --cov-report=xml
```

Comment:
I think it's simpler to have server tests and local tests -- we may end up not even running the server ones on every linux case.